### ISSUE TITLE
aar file from maven repo contains app icons

### ISSUE THREAD

**Issue by claust** [RomainPiel/Shimmer-android#23]
Even though fixed in the Github code, the 1.4.0 aar file that you download from maven.org still contains the ic_launcher.png files.

To verify, please download the aar file directly, unzip it, and see the ic_launcher.png files inside:
http://central.maven.org/maven2/com/romainpiel/shimmer/library/1.4.0/library-1.4.0.aar

Original issue is here: https://github.com/RomainPiel/Shimmer-android/issues/20

**Comment by fimh** [comment]
Yes, claust was right. I also encountered the same problem in 1.4.0.

**Comment by Jaymo** [comment]
Easiest way to solve this is by adding  tools:replace="android:icon" in the application tag of your manifest

**Comment by RomainPiel** [comment]
Sorry guys I don't have much time to spend on shimmer at the moment. I'll
try to spend some time this weekend but I can't promise anything.

**Comment by vhenriques** [comment]
@Jaymo 

tools:replace="android:icon"

didnt work

**Comment by casperbang** [comment]
I also ran into this issue, the ic_launcher was removed from the repository itself but a 1.5 Maven release would be beneficial.

**Comment by RomainPiel** [comment]
I don't have much time to spend on Shimmer anymore and it's not a very complex library. Feel free to fork or copy the classes you need into your project.
